pkg/api/v1/cluster_types.go

```diff
@@ -110,17 +119,20 @@ type RepairTaskSpec struct {
 	// For Scylla clusters that *do not support row-level repair*, intensity can be a decimal between (0,1).
 	// In that case it specifies percent of shards that can be repaired in parallel on a repair master node.
 	// For Scylla clusters that are row-level repair enabled, setting intensity below 1 has the same effect as setting intensity 1.
+	// +kubebuilder:default:="1"
 	Intensity *string `json:"intensity,omitempty" mapstructure:"intensity,omitempty"`
 	// Parallel the maximum number of Scylla repair jobs that can run at the same time (on different token ranges and replicas).
 	// Each node can take part in at most one repair at any given moment. By default the maximum possible parallelism is used.
 	// The effective parallelism depends on a keyspace replication factor (RF) and the number of nodes.
 	// The formula to calculate it is as follows: number of nodes / RF, ex. for 6 node cluster with RF=3 the maximum parallelism is 2.
+	// +kubebuilder:default:=0
```

Comment:
we can probably get rid of the pointers globally, as you have already done somewhere, when there is a default